pipelinewise/fastsync/commons/target_snowflake.py

```diff
@@ -60,12 +60,18 @@ def open_connection(self):
                                            autocommit=True,
                                            session_parameters={
                                                # Quoted identifiers should be case sensitive
-                                               'QUOTED_IDENTIFIERS_IGNORE_CASE': 'FALSE'
+                                               'QUOTED_IDENTIFIERS_IGNORE_CASE': 'FALSE',
+                                               'QUERY_TAG': json.dumps({
+                                                   'ppw_component': 'fastsync',
+                                                   'tap_id': self.connection_config.get('tap_id'),
+                                                   'table': query_tag_props.get('schema'),
```

Comment:
`query_tag_props` can be `None`, calling `get` on it would fail.

Reply:
Empty dictionary could be a better default value of `query_tag_props`

Reply:
fixed by 26febe82084667ac7e7db47299147e0dfa284257